README.md

```diff
@@ -267,6 +268,87 @@ rows-any
 
 </details>
 
+<details>
+<summary><strong>DENSIFYROWS</strong></summary>
+
+### DENSIFYROWS
+
+**Description**
+
+```
+Removes empty or incomplete rows from sparse data. This is a convenience wrapper around DENSIFY that specifically targets row operations.
+```
```

Comment:
Be slightly more specific, so that if we build a DENSIFYROWSANY, that the description will distinguish between the two.